client/src/components/ReviewTile.js

```diff
@@ -1,20 +1,107 @@
-import React from "react";
+import React, { useState, useEffect } from 'react'
+import { withRouter } from 'react-router-dom'
 
-const ReviewTile = ({ title, rating, body, recipe }) => {
-  let reviewDescription;
+const ReviewTile = ({ title, rating, body, recipe, user, id, voteCount, reviewersVoted }) => {
+
+  let reviewDescription
   if (body) {
     reviewDescription = <p>Review: {body}</p>;
   }
+
+  const [totalVotes, setTotalVotes] = useState(voteCount)
+  const [userVoted, setUserVoted] = useState(false)
+  
+  const getVotes = async () => {
+    try {
+      const response = await fetch("/api/v1/votes")
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      }
+      const responseBody = await response.json()
+      setTotalVotes(totalVotes, responseBody.value)
+    } catch (error) {
+      return console.error(`Error in fetch: ${error.message}`)
+    }
+  }
```

Comment:
unfortunately, like the reviews, you should be getting all of the votes for your reviews when you make your initial get on the show component. 

Imagine if there were 20 reviews on the page. Thats 20 different requests for reviews!!!
Versus one request for everything. Which means....BACK TO THE SERIALIZER!